### PR TITLE
Doc: Clarify PVC access modes in third paragraph

### PR DESCRIPTION
**Pull request by manderson-it**
The last sentence of the third paragraph was worded in an unreadable way.
Before: Claims can request specific size and access modes (e.g., they can be mounted once read/write or many times read-only).
Replaced with technical terms and hyperlinked to Access Modes section on the same page.

### CODE REVIEW

**Comment by manderson-it** [comment]
I signed it

**Comment by msau42** [comment]
/lgtm

Unrelated note: I'm unsure about the analogy to Pod and Node. PVC and PV are one to one bindings which is different from the Pod-Node mapping.

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Managing storage is a distinct problem from managing compute instances. The Pers
...
 
-A _PersistentVolumeClaim_ (PVC) is a request for storage by a user. It is similar to a Pod. Pods consume node resources and PVCs consume PV resources. Pods can request specific levels of resources (CPU and Memory).  Claims can request specific size and access modes (e.g., they can be mounted once read/write or many times read-only).
+A _PersistentVolumeClaim_ (PVC) is a request for storage by a user. It is similar to a Pod. Pods consume node resources and PVCs consume PV resources. Pods can request specific levels of resources (CPU and Memory).  Claims can request specific size and access modes (e.g., they can be mounted ReadWriteOnce, ReadOnlyMany or ReadWriteMany, see [AccessModes](#access-modes)).
```
Comment (tengqm) [inline content/en/docs/concepts/storage/persistent-volumes.md:33]: > Claims can request specific size and access modes (e.g., they can be mounted ReadWriteOnce, ReadOnlyMany or ReadWriteMany, see [AccessModes](#access-modes)).

Claims can request specific size and access modes. For example, they can request to be mounted as ReadWriteOnce, ReadOnlyMany or ReadWriteMany. See [Access Modes](#access-modes) for more details.

**Pull request merged**